.github/workflows/cypress-firefox.yml

```
@@ -0,0 +1,22 @@
+name: Cypress push and pull
+
+on: [push, pull_request]
+
+jobs:
+  cypress-run:
+     name: firefox
+     runs-on: ubuntu-18.04
```

These lines are indented just a bit too much:

```suggestion
    name: firefox
    runs-on: ubuntu-18.04
```